sdk/boms/azure-spring-boot-bom/pom.xml

```diff
@@ -38,7 +38,7 @@
     <packageOutputDirectory>${project.build.directory}</packageOutputDirectory>
     <legal><![CDATA[[INFO] Any downloads listed may be third party software.  Microsoft grants you no rights for third party software.]]></legal>
     <azure.spring.boot.integration.version>${project.version}</azure.spring.boot.integration.version>
-    <azure.spring.boot.jms.integration.version>2.3.4</azure.spring.boot.jms.integration.version>
+    <azure.spring.boot.storage.integration.version>1.2.8</azure.spring.boot.storage.integration.version>
```

Comment:
`azure.spring.boot.jms.integration.version` is still used in this file. For example line 62 and line 67

Reply:
done